libnetwork/controller.go

```diff
@@ -341,7 +341,7 @@ func (c *Controller) makeDriverConfig(ntype string) map[string]interface{} {
 	}
 
 	if c.cfg.Scope.IsValid() {
-		// FIXME: every driver instance constructs a new DataStore
+		// FIXME: every driver instance constructs a new FoobarStore
```

Comment:
Whoops. I used `FoobarStore` during the rename. I'll fix it back before merging 😂